### ISSUE TITLE
resourceop example fails as secrets creation is forbidden

### ISSUE THREAD

**Issue by ryandawsonuk** [kubeflow/pipelines#2623]
The [ResourceOp example](https://github.com/kubeflow/pipelines/blob/master/samples/core/resource_ops/resource_ops.py) creates a Secret. But it can't do that in a cluster when you do a kubeflow install as pipeline-runner doesn't have that permission by default.

```
This step is in Failed state with this message: Error from server (Forbidden): error when creating "/tmp/manifest.yaml": secrets is forbidden: User "system:serviceaccount:kubeflow:pipeline-runner" cannot create resource "secrets" in API group "" in the namespace "kubeflow"
```
Does it need to be a Secret that is created?

**Comment by numerology** [comment]
Hi @ryandawsonuk . Can you please upgrade KFP to the latest version? It's been changed to Job instead of secret. Thanks!

**Comment by numerology** [comment]
Fixed by #2274

**Comment by ryandawsonuk** [comment]
Thanks for the quick turnaround
